internal/legacy/signals/signals_windows.go

```diff
@@ -0,0 +1,96 @@
+//go:build windows
+// +build windows
+
+// Note: this build on windows systems
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signals
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// ExitSignals are used to determine if an incoming os.Signal should cause termination.
+	ExitSignals = map[os.Signal]bool{
+		syscall.SIGHUP:  true,
+		syscall.SIGINT:  true,
+		syscall.SIGABRT: true,
+		syscall.SIGILL:  true,
+		syscall.SIGQUIT: true,
+		syscall.SIGTERM: true,
+		syscall.SIGSEGV: true,
+		//syscall.SIGTSTP: true,
+	}
+	// ExitChannel is for gracefully terminating the LogSignals() function.
+	ExitChannel = make(chan bool, 1)
+	// SignalChannel is for listening to OS signals.
+	SignalChannel = make(chan os.Signal, 1)
+	// Debug is defined globally for mocking logrus.Debug statements.
+	Debug func(args ...interface{}) = logrus.Debug
+)
+
+// Watch concurrently logs debug statements when encountering interrupt signals from the OS.
+// This function relies on the os/signal package which may not capture SIGKILL and SIGSTOP.
+func Watch() {
+	Debug("Watching for OS Signals...")
+	// Observe all signals, excluding SIGKILL and SIGSTOP.
+	signal.Notify(SignalChannel)
+	// Continuously log signals.
+	go LogSignals()
+}
+
+// Stop gracefully terminates the concurrent signal logging.
+// TODO: @tylerferrara Currently we don't gracefully exit, since the Auditor is designed
```

Comment:
please drop the user and create an issue and add that here

Reply:
done